gnomad_constraint/utils/constraint.py

```diff
@@ -386,3 +392,171 @@ def apply_models(
     )
     # Compute the observed:expected ratio.
     return ht.annotate(obs_exp=ht.observed_variants / ht.expected_variants)
+
+
+def compute_constraint_metrics(
+    ht: hl.Table,
+    keys: Tuple[str] = ("gene", "transcript", "canonical"),
+    classic_lof_annotations: Set[str] = {
+        "stop_gained",
+        "splice_donor_variant",
+        "splice_acceptor_variant",
+    },
+    pops: Tuple[str] = (),
+    expected_values: Dict[str, float] = {"Null": 1.0, "Rec": 0.463, "LI": 0.089},
```

Comment:
In the PR I changed this to a default of None, but if None it actually uses `{"Null": 1.0, "Rec": 0.463, "LI": 0.089}` so that is still technically the default. I was getting a warning since the default is mutable.